src/main/webapp/index.html

```diff
@@ -26,69 +26,67 @@ <h1>Census Viz</h1>
+            onkeyup="validateInput('action')" onfocus="storeValueAndEmpty('action')"
+            onfocusout="replaceValueIfEmpty('action')" value="live in" pattern="live in|work in|moved to">
+          <datalist id="action">
+            <option data-value="live" value="live in"></option>
+            <option data-value="work" value="work in"></option>
+            <option data-value="moved" value="moved to"></option>
+          </datalist>
           </input>
 
-          <input type="list" list="location" name="location" id="location-list"
-              autocomplete="off" required onkeyup="validateInput('location')"
-              onfocus="storeValueAndEmpty('location')"
-              onfocusout="replaceValueIfEmpty('location')"
-              value="each U.S. state">
-            <datalist id="location"></datalist>
+          <input type="list" list="location" name="location" id="location-list" autocomplete="off" required
+            onkeyup="validateInput('location')" onfocus="storeValueAndEmpty('location')"
+            onfocusout="replaceValueIfEmpty('location')" value="each U.S. state">
+          <datalist id="location"></datalist>
           </input>
           in
-          <input type="list" list="year" name="year" id="year-list"
-              autocomplete="off" required onkeyup="validateInput('year')"
-              onfocus="storeValueAndEmpty('year')"
-              onfocusout="replaceValueIfEmpty('year')"
-              value="2018" pattern="201[0-8]">
-            <datalist id="year">
-              <option value="2010"></option>
-              <option value="2011"></option>
-              <option value="2012"></option>
-              <option value="2013"></option>
-              <option value="2014"></option>
-              <option value="2015"></option>
-              <option value="2016"></option>
-              <option value="2017"></option>
-              <option value="2018"></option>
-            </datalist>
+          <input type="list" list="year" name="year" id="year-list" autocomplete="off" required
+            onkeyup="validateInput('year')" onfocus="storeValueAndEmpty('year')"
+            onfocusout="replaceValueIfEmpty('year')" value="2018" pattern="201[0-8]">
+          <datalist id="year">
+            <option value="2010"></option>
+            <option value="2011"></option>
+            <option value="2012"></option>
+            <option value="2013"></option>
+            <option value="2014"></option>
+            <option value="2015"></option>
+            <option value="2016"></option>
+            <option value="2017"></option>
+            <option value="2018"></option>
+          </datalist>
           </input>
           ?
         </p>
         <input type="submit" value="go">
       </form>
     </div>
     <div id="result">
+      <div id="map-options">
+        <div class="map-option">Flat map</div>
```

Comment:
The name "Flat map" makes a lot of sense to me. But then "map overlay" doesn't make as much sense, because I don't know what's being overlaid (the word "map" is already being used to mean the other thing).
What about "Flat view" vs "Map overlay" or "Map view"? That way we save the word map for just the geoJson.
Or, what about "Flat map" vs "Terrain"? Terrain isn't exactly right to describe what the geoJson shows, but it's pretty close.

Reply:
Changed to "Flat view" and "Map overlay"

Reply:
Late but I like these names